tests/sentry/api/endpoints/test_project_tagkey_values.py

```diff
@@ -33,11 +45,21 @@ def test_simple(self):
         assert response.data[0]["value"] == "bar"
 
     def test_query(self):
-        project = self.create_project()
-        tagkey = tagstore.create_tag_key(project_id=project.id, environment_id=None, key="foo")
-        tagstore.create_tag_value(
-            project_id=project.id, environment_id=None, key="foo", value="bar"
-        )
+        if settings.SENTRY_TAGSTORE in [
+            "sentry.tagstore.snuba.SnubaCompatibilityTagStorage",
+            "sentry.tagstore.snuba.SnubaTagStorage",
+        ]:
```

Comment:
Should we start warning folks when they are using a tagstore that isn't one of these?

Reply:
What kind of warning are you thinking of? I was planning to just completely delete Legacy immediately after this gets merged (v2 is already gone)